lib/importer/src/index.js

```diff
@@ -171,7 +175,16 @@ exports.Initialise = (config, router, prototypeKit) => {
 
       if (session.sheets.length == 1) {
         session.sheet = session.sheets[0];
+
+        // When there is only a single sheet, and if the prototype is configured to
+        // automatically progress, then this will do so if the 'other' url is
+        // specified.
+        if (plugin_config.sheetSelection.toLowerCase() == "automatic" && "other" in request.query) {
+          const otherPage = decodeURIComponent(request.query.other)
+          if (otherPage) { request.query.next = request.query.other }
```

Comment:
Does this need to come after the line below that "ensures the session is persisted"?